### PR TITLE
Add UI for Tags and Update UG for Tags

### PR DESCRIPTION
**Pull request by LuWenQ**
added Tags UI and change UG for tag
fixed help command

### CODE REVIEW

**Review by le0tan — APPROVED** [review]
Looks good to me.

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -205,19 +205,19 @@ Deletes the 2nd find rule in the displayed list.
...
 Examples:
 
 * `newtag n/sssp`
 
-==== Listing tags: `lstag`
+==== Listing tags: `listtag`
 
 Displays a list of all existing tags. +
-Format: `lstag`
+Format: `liststag`
```
Comment (Seris370) [inline docs/UserGuide.adoc]: typo here

File: src/main/java/seedu/algobase/logic/commands/ListPlanCommand.java [hunk 2]
```diff
@@ -13,6 +13,11 @@
 
     public static final String COMMAND_WORD = "listplan";
 
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays a list of all existing plans alphabetically.\n"
```
Comment (Seris370) [inline src/main/java/seedu/algobase/logic/commands/ListPlanCommand.java]: I thought plans were displayed in chronological order...?

**Review by Seris370 — COMMENTED** [review]
LGTM.

**Pull request merged**